builtin/rebase--helper.c

```diff
@@ -83,8 +111,28 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		return !!sequencer_continue(&opts);
 	if (command == ABORT && argc == 1)
 		return !!sequencer_remove_state(&opts);
-	if (command == MAKE_SCRIPT && argc > 1)
-		return !!sequencer_make_script(stdout, argc, argv, flags);
+	if (command == MAKE_SCRIPT && (argc == 1 || argc == 2)) {
+		struct argv_array f_argv = ARGV_ARRAY_INIT;
+
+		if (!upstream && squash_onto)
+			write_file(path_squash_onto(), "%s\n", squash_onto);
+
+		ret = get_revision_ranges(upstream, onto, &head_hash, &revisions);
+		if (ret)
+			return ret;
+
+		argv_array_push(&f_argv, argv[0]);
+		argv_array_push(&f_argv, revisions);
+		if (argc == 2)
+			argv_array_push(&f_argv, argv[1]);
+
+		ret = !!sequencer_make_script(stdout, f_argv.argc, f_argv.argv, flags);
```

Comment:
Okay, I made up my mind: `make_script_args` is my favorite alternative name for `f_argv`.